challenge_help_me_carry/src/challenge_hmc_functions/hmc_functions.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/python
+import roslib;
+import rospy
+import smach
+import sys
+import time
+import datetime
+from hmi import TimeoutException
+
+from cb_planner_msgs_srvs.msg import PositionConstraint
+
+import robot_smach_states.util.designators as ds
+from robot_smach_states.util.designators import check_type
+from robot_skills.arms import Arm, GripperState
+
+import robot_smach_states as states
+
+from robocup_knowledge import load_knowledge
+from robot_skills.util import kdl_conversions
+from robot_skills.util.entity import Entity
+
+challenge_knowledge = load_knowledge('challenge_help_me_carry')
+
+def setup_challenge(self, robot):
+
```

Comment:
Starting a function with an empty line seems weird, if you want more separation, add one before the function, but likely there is a Python ROS style guide (ie pep8) that defines what you may do here.